src/main/java/com/redhat/darcy/webdriver/WebDriverBrowser.java

```diff
@@ -174,6 +176,11 @@ public void closeAll() {
     }
 
     @Override
+    public File takeScreenshot() {
+        return driver.getScreenshotAs(OutputType.FILE);
```

Comment:
Need to wrap in an `attemptAndGet` in case the underlying `TargetedWebDriver` is not actually present. I would add a unit test to see what I mean. Have the underlying driver throw NoSuchWindowException on `getScreenshotAs`, the expected exception should be a darcy `FindableNotPresentException`. We wouldn't want to expose underlying API (hence why everything else wraps in try/catch).